Source/Scene/GroupMetadata.js

```diff
@@ -127,16 +127,31 @@ Object.defineProperties(GroupMetadata.prototype, {
 });
 
 /**
- * Returns whether this property exists.
+ * Returns whether the group has this property.
  *
  * @param {String} propertyId The case-sensitive ID of the property.
- * @returns {Boolean} Whether this property exists.
+ * @returns {Boolean} Whether the group has this property.
  * @private
  */
 GroupMetadata.prototype.hasProperty = function (propertyId) {
   return MetadataEntity.hasProperty(propertyId, this._properties, this._class);
 };
 
+/**
+ * Returns whether the group has a property with the given semantic.
+ *
+ * @param {String} semantic The case-sensitive semantic of the property.
+ * @returns {Boolean} Whether the group has a property with the given semantic.
+ * @private
+ */
+GroupMetadata.prototype.hasPropertyBySemantic = function (semantic) {
```

Comment:
Hm, which terminology looks nicer of the two?

* `hasPropertyBySemantic`/`propertyExistsBySemantic`
* or `hasSemantic`/`semanticExists`

The latter sounds more natural, but the former has nicer symmetry with the other methods. Not sure which is better.

Reply:
I debated this as well. I think we should be consistent, and I'm ok with the more concise version everywhere.